SQL_2709_Date_Time.sql

```diff
@@ -0,0 +1,111 @@
+----------------DATE AND TIME FUNCTIONS---------------
+
+
+--37. Write a query to display the current date and time.
+SELECT GETDATE();
+
+--38. From an 'Orders' table, find all orders placed in the last 24 hours.
+SELECT * FROM Orders WHERE OrdDate >= DATEADD(HOUR,-24,GETDATE());
+
+--39. Display the current UTC date and time.
+SELECT GETUTCDATE();
+
+--40. Write a query to show the time difference between local time and UTC time.
+SELECT DATEDIFF(MINUTE,GETDATE(), GETUTCDATE());
+
+--41. Convert the current date and time to 'Pacific Standard Time'.
+SELECT GETDATE() AT TIME ZONE 'Pacific Standard Time';

```

Comment:
(https://learn.microsoft.com/en-us/sql/t-sql/queries/at-time-zone-transact-sql?view=sql-server-ver16#b-convert-values-between-different-time-zones)

First we need to specify the time zone of our stored data, then specify the target time zone.